node/chainSimulator/configs/configs.go

```diff
@@ -95,16 +102,14 @@ func CreateChainSimulatorConfigs(args ArgsChainSimulatorConfigs) (*ArgsConfigsSi
 		return nil, err
 	}
 
-	gasScheduleName, err := GetLatestGasScheduleFilename(configs.ConfigurationPathsHolder.GasScheduleDirectoryName)
-	if err != nil {
-		return nil, err
-	}
-
 	configs.GeneralConfig.SmartContractsStorage.DB.Type = string(storageunit.MemoryDB)
 	configs.GeneralConfig.SmartContractsStorageForSCQuery.DB.Type = string(storageunit.MemoryDB)
 	configs.GeneralConfig.SmartContractsStorageSimulate.DB.Type = string(storageunit.MemoryDB)
 
-	maxNumNodes := uint64(args.MinNodesPerShard*args.NumOfShards+args.MetaChainMinNodes) + uint64(args.NumOfShards+1)
+	maxNumNodes := uint64((args.MinNodesPerShard+args.NumNodesWaitingListShard)*args.NumOfShards) +
+		uint64(args.MetaChainMinNodes+args.NumNodesWaitingListMeta) +
+		2*uint64(args.NumOfShards+1+args.NumNodesWaitingListShard+args.NumNodesWaitingListMeta)
```

Comment:
do we still need this 2*.... part?

Reply:
yes in this case because we want  the maximum number of nodes to be greater sum of all waiting and eligible nodes

Reply:
why?